### PR TITLE
More explicite message for Exception

### PR DESCRIPTION
**Pull request by chadyred**
Hello,

I'm always doing this when an error occur with the serialization.

Thanks for your works

### CODE REVIEW

File: src/Type/ODMType.php [hunk 1]
```diff
@@ -113,7 +113,7 @@ public function convertToDatabaseValue($value, AbstractPlatform $platform): ?str
             $context = $this->getSerializationContext();
             $value   = $this->getSerializer()->serialize($value, $this->format, $context);
         } catch (ExceptionInterface $exception) {
-            $message = sprintf('Serialization exception occurred for class "%s".', $this->getEntityClass());
+            $message = sprintf('Serialization exception occurred for class "%s". (%s)', $this->getEntityClass(), $exception->getMessage());
```
Comment (strider2038) [inline src/Type/ODMType.php]: Hello! Thanks for PR. I'm suggesting another format.

```suggestion
            $message = sprintf('Serialization exception occurred for class "%s": %s.', $this->getEntityClass(), $exception->getMessage());
```

File: src/Type/ODMType.php [hunk 2]
```diff
@@ -134,7 +134,7 @@ public function convertToPHPValue($value, AbstractPlatform $platform): ?object
             $context = $this->getDeserializationContext();
             $value   = $this->getSerializer()->deserialize($value, $this->getEntityClass(), $this->format, $context);
         } catch (ExceptionInterface $exception) {
-            $message = sprintf('Deserialization exception occurred for class "%s".', $this->getEntityClass());
+            $message = sprintf('Deserialization exception occurred for class "%s". (%s)', $this->getEntityClass(), $exception->getMessage());
```
Comment (strider2038) [inline src/Type/ODMType.php]:
```suggestion
            $message = sprintf('Deserialization exception occurred for class "%s": %s.', $this->getEntityClass(), $exception->getMessage());
```

**Review by strider2038 — CHANGES_REQUESTED** [review]
Please, update the tests. I can't merge until all tests pass.

**Comment by chadyred** [comment]
A test won't pass and I don't know why it doesn't find a package : https://github.com/goodwix/doctrine-json-odm/runs/5462396435?check_suite_focus=true

**Pull request merged**